cmd/finch/nerdctl.go

```diff
@@ -121,10 +126,27 @@ func (nc *nerdctlCommand) run(cmdName string, args []string) error {
 				arg = fmt.Sprintf("%s%s", arg[0:11], resolvedIP)
 			}
 			nerdctlArgs = append(nerdctlArgs, arg)
+		case strings.HasPrefix(arg, "-v") || strings.HasPrefix(arg, "--volume"):
+			if dockerCompat {
```

Comment:
Is this temporary change? If yes, can we add a todo to ensure we don't forget to remove this section. Maybe we can add another issue and add the link here.

Reply:
Good call. I will put nerdctl issue here and keep the Finch issue open.